### ISSUE TITLE
Too many redirects.

### ISSUE THREAD

**Issue by Fangyh09** [anibali/h36m-fetch#1]
It says too many redirects. Which version of axel do you use?
```
Initializing download: http://vision.imar.ro/human3.6m/filebrowser.php?download=1&filepath=Poses/D2_Positions&filename=SubjectSpecific_3.tgz
Too many redirects.
```

**Comment by saketdingliwal** [comment]
Did you find a solution? I am facing the same problem. Its giving the same error : Too many redirects.

**Comment by saketdingliwal** [comment]
Yes I have edited the file name to 'config.ini' and added the PHPSESSID corresponding to my account. Here take a look at the screenshot.

![screenshot from 2018-10-27 15-10-36](https://user-images.githubusercontent.com/16961778/47602299-9c2c4b00-d9fa-11e8-909e-2b7827d90b24.png)

the rest of the x's are also replaced by my PHPSESSID. I just hid them for privacy concerns

**Comment by anibali** [comment]
What you've done is correct. The other possibility is that you're currently logged out of the Human3.6M website. Go to http://vision.imar.ro/human3.6m/ and make sure that you're logged in.

**Comment by anibali** [comment]
I have written https://github.com/anibali/h36m-fetch/commit/098cf6cc27e5d41ab35a76d8cbaa9869e4810162 which verifies the PHPSESSID before attempting to download files. Hopefully this will provide better feedback for future users that have an invalid PHPSESSID or are logged out.

**Comment by prakharg24** [comment]
Me and @saketdingliwal were working on it together. Yes you are right, we were logged out from Human3.6M website at that moment. The script is working fine now. It's a good thing you updated the script to display a different error message. This will help people in future. Thank You

**Comment by guofuzheng** [comment]
Hello, i have created an account on Human3.6m, but when i want to sign in, it always show "Account needs manual verification from the admins. Please be patient.".So, may i share your dataset with you?
